### DISCUSSION TITLE
Disabled Feature

### DISCUSSION THREAD

**Discussion by CaamVilvactDJavu** [LazyVim/LazyVim#3091] [General]
![Screenshot 2024-05-02 181333](https://github.com/LazyVim/LazyVim/assets/65745056/ba55f2ae-6b21-4fd7-951b-29a3d4012888)

How do I turn off the following feature where at the top of the code (18, 16, 15) is the code contained in the code marked by my cursor!

**Comment by dpetka2001** [comment]
It's [nvim-treesitter-context](https://www.lazyvim.org/plugins/treesitter#nvim-treesitter-context). You can either disable the plugin as a whole or you can toggle the feature off with `<leader>ut` for the running Neovim session.

**Reply by CaamVilvactDJavu** [reply]
Thank You @dpetka2001, It's work.
